source/_integrations/harmony.markdown

```diff
@@ -32,6 +32,8 @@ Supported units:
 
 {% include integrations/config_flow.md %}
 
+**Note:** Depending on the firmware, you may need to enable XMPP for this integration to work.
```

Comment:
Maybe a note on how to do it?

From your Harmony app, go to: Menu > Harmony Setup > Add/Edit Devices & Activities > Remote & Hub > Enable XMPP

Reply:
Updated